build.sh

```diff
@@ -331,21 +341,64 @@ cp $dist_dir/$ffmpeg_target/ffmpeg$exe_extension \
 if [ ! $skip_packaging == 1 ]; then
 
   log "Packaging v$meta_version for $target"
-
+  # ===============================================
+  # LINUX
+  # ===============================================
   if [ $target_os == "linux" ]; then
     mkdir -p $target_dist_dir/deb/opt/$package_binary_name
     mkdir -p $target_dist_dir/deb/DEBIAN
+    # --------------------------------
+    # Variation: No ffmpeg shipped
+    # --------------------------------
+    cp LICENSE.txt README.md app/node_modules/open/xdg-open \
+      $target_dist_dir/$package_binary_name \
+      $target_dist_dir/deb/opt/$package_binary_name
+
+    yq ".package.deb" ./config.toml -o yaml | \
+      yq e ".package = \"${meta_id}.noffmpeg\"" |\
+      yq e ".conflicts = \"${meta_id}\"" |\
+      yq e ".description = \"${meta_description} (No pre-built ffmpeg binary shipped; use ffmpeg on system instead)\"" |\
```

Comment:
> (No pre-built ffmpeg binary shipped; use ffmpeg on system instead)

(No pre-built ffmpeg binary shipped; relies on system ffmpeg)

Reply:
OK, replaced with the new sentence.